azooKeyMac.xcodeproj/project.pbxproj

```diff
@@ -491,22 +491,25 @@
 				CURRENT_PROJECT_VERSION = 1;
 				DEAD_CODE_STRIPPING = YES;
 				DEVELOPMENT_ASSET_PATHS = "\"azooKeyMac/Preview Content\"";
-				DEVELOPMENT_TEAM = 9S3UXHYP65;
+				DEVELOPMENT_TEAM = 23PW34QDKR;
```

Comment:
I think this kind of diff is not necessary for this PR. Please keep the original version.